evm/p2p/constants.py

```diff
@@ -0,0 +1,25 @@
+supported_rlpx_version = 4
+
+# Overhead added by ECIES encryption
+encrypt_overhead_length = 113
+
+# Lentgh of elliptic S256 signatures
+signature_len = 65
+
+# Length of public keys: 512 bit keys in uncompressed form, without format byte
+pubkey_len = 64
+
+# Hash length (for nonce etc)
+hash_len = 32
+
+# Length of initial auth handshake message
+auth_msg_len = signature_len + hash_len + pubkey_len + hash_len + 1
+
+# Length of auth ack handshake message
+auth_ack_len = pubkey_len + hash_len + 1
+
+# Length of encrypted pre-EIP-8 initiator handshake
+encrypted_auth_msg_len = auth_msg_len + encrypt_overhead_length
+
+# Length of encrypted pre-EIP-8 handshake reply
+encrypted_auth_ack_len = auth_ack_len + encrypt_overhead_length
```

Comment:
Can all of these be changed to `CAP_CASE` so that it's a bit easier to tell they are constants when they are used in other modules.